pkg/scheduler/plugins/coscheduling/core/core.go

```diff
@@ -299,10 +303,26 @@ func (pgMgr *PodGroupManager) PreFilter(ctx context.Context, _ *framework.CycleS
 	return nil
 }
 
-// PostFilter
+// PostFilter invoked at the postFilter extension point.
+func (pgMgr *PodGroupManager) PostFilter(ctx context.Context, state *framework.CycleState, pod *corev1.Pod, m framework.NodeToStatusMap) (*framework.PostFilterResult, *framework.Status) {
+	if !pointer.BoolDeref(pgMgr.args.EnablePreemption, false) {
```

Comment:
Recommended to set the default in plugin args, instead of getting a value at runtime.